pkgs/applications/video/shotcut/default.nix

```diff
@@ -34,6 +35,13 @@ stdenv.mkDerivation rec {
     sed "s_/usr/bin/nice_''${NICE}_" -i src/jobs/meltjob.cpp src/jobs/ffmpegjob.cpp
   '';
 
+  patchFlags = "-p0";
+  patches = [ (fetchpatch {
+    url = https://github.com/mltframework/shotcut/files/2075592/shotcut-18.06.02-qt.patch.txt;
```

Comment:
This url is stable (does not change its content)? Is `2075592` a tag or something similar?

Reply:
Don't know. Changed it.